kafka/conn.py

```diff
@@ -486,9 +481,6 @@ def _try_handshake(self):
         # old ssl in python2.6 will swallow all SSLErrors here...
         except (SSLWantReadError, SSLWantWriteError):
```

Comment:
Should this also check whether python < 2.7 before `pass`'ing? Just in the off-chance these exceptions get-used later?

Reply:
no, see the import hackery above. this is an old problem. we could try to check exception error codes etc, but unless there's a real user asking for help w/ old python installs I'm not that interested in adding more compatibility layers here.

Plus python 2.7 is EOL very soon...